# Update the .desktop file to match the current FreeDesktop.org specifications.
# Add Catalan and Spanish translations of the comment.
# Execute fusion-icon with the "--no-start" option.

diff -Nur fusion-icon-0.0.0+git20071028/fusion-icon.desktop fusion-icon-0.0.0+git20071028.new/fusion-icon.desktop
--- fusion-icon-0.0.0+git20071028/fusion-icon.desktop	2007-12-11 21:23:54.000000000 +0100
+++ fusion-icon-0.0.0+git20071028.new/fusion-icon.desktop	2007-12-11 22:21:15.000000000 +0100
@@ -1,9 +1,11 @@
 [Desktop Entry]
+Version=1.0
 Name=Compiz Fusion Icon
 Comment=Start and manage Compiz Fusion
-Exec=fusion-icon
+Comment[ca]=Gestioneu el Compiz Fusion i trieu el gestors de finestres a utilitzar
+Comment[es]=Configura el Compiz Fusion y cambia entre los distintos gestores de ventanas instalados
+Exec=fusion-icon --no-start
 Icon=fusion-icon
 Terminal=false
 Type=Application
-Categories=System;Application;
-Encoding=UTF-8
+Categories=System;
